arch/risc-v/src/mpfs/mpfs_start.c

```diff
@@ -65,6 +65,47 @@ volatile bool g_serial_ok = false;
 
 extern void mpfs_cpu_boot(uint32_t);
 
+#ifdef CONFIG_MPFS_BOOTLOADER
+
+extern void mpfs_opensbi_prepare_hart();
+
+/* Default boot address for every hart */
+
+uint64_t g_entrypoints[5] =
```

Comment:
```suggestion
uintptr_t g_entrypoints[5] =
```
here and in other places?

Reply:
probably can simply do

```
extern uintptr_t mpfs_opensbi_prepare_hart;

uint64_t *g_entrypoints[5] =
{
#ifdef CONFIG_MPFS_HART0_SBI
  &mpfs_opensbi_prepare_hart,
#else
  (uintptr_t *)CONFIG_MPFS_HART0_ENTRYPOINT,
#endif
...
}

```

Reply:
why? I know that this is uint64_t on the mpfs platform. also, other places are not really subject to this PR